packages/xrpl/test/integration/transactions/payment.test.ts

```diff
@@ -12,9 +14,20 @@ const TIMEOUT = 20000
 
 describe('Payment', function () {
   let testContext: XrplIntegrationTestContext
+  let paymentTxExample: Payment
```

Comment:
```suggestion
  let paymentTx: Payment
```
To keep it simpler.

Reply:
updated in [28d1ef4](https://github.com/XRPLF/xrpl.js/pull/2689/commits/28d1ef47d738527dc4d4774c81629f0bae8dc521)